lib/cogctl/actions/users/info.ex

```diff
@@ -1,41 +1,50 @@
 defmodule Cogctl.Actions.Users.Info do
   use Cogctl.Action, "users info"
-  alias Cogctl.Table
+  import Cogctl.Actions.Users.Util, only: [render: 4]
 
   def option_spec do
     [{:user, :undefined, :undefined, {:string, :undefined}, 'User username (required)'}]
   end
 
+  def run(options, _args, _config, %{token: nil}=endpoint),
+    do: with_authentication(endpoint, &run(options, nil, nil, &1))
```

Comment:
Is this ever legitimately called with `endpoint` arguments that have a token and with one that doesn't? If not, this construction is a little awkward to read. For instance, it took me a second to realize that "No, we didn't just accidentally stop authenticating this API endpoint call".